runtime/scripts.vim

```diff
@@ -245,7 +245,7 @@ else
     set ft=xhtml
 
     " HTML (e.g.: <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN")
-  elseif s:line1 =~? '\<DOCTYPE\s\+html\>'
+  elseif s:line1 =~? '\<<!DOCTYPE\s\+html\>'
```

Comment:
@k-takata You're right. Not sure why my manual tests passed before. I guess `<!\<DOCTYPE\s+html\>` is more correct?